device-detector/tests/test_main.py

```
@@ -0,0 +1,293 @@
+        
+        # When: runを実行
+        service.run()
+        
+        # Then: 停止される
+        mock_stop.assert_called_once()
+    
+    def test_main_entry_point(self):
+        """__main__エントリーポイントのテスト"""
+        # Given: __name__ == '__main__'のシミュレーション
+        with patch('src.main.__name__', '__main__'):
+            with patch('src.main.main') as mock_main:
```

Flake8 (7.2.0)

[error] local variable 'mock_main' is assigned to but never used

(F841)